spot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 5

  # Position Controllers ---------------------------------------
  leg_fl_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg_fl_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  leg_rl_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg_rl_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  leg_fr_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg_fr_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  leg_rr_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg_rr_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
